ui/site/src/util.js

```diff
@@ -42,7 +49,7 @@ lichess.storage = (function() {
         },
         listen: function(f) {
           window.addEventListener('storage', function(e) {
-            if (e.key === k && e.newValue !== null) f(e);
+            if (e.key === k && e.newValue !== null && e.newValue !== e.oldValue) f(e);
```

Comment:
but this code is used in other places. Did you check that this functional change does not break other users of `listen`?